group.html

```diff
@@ -317,7 +317,56 @@ <h3>Ms. Kosar Yasin</h3>
         </div> 
       </div> <!-- Bio & Research -->
       <br>
-      <!-- END OF: Graduate Student: Kosar Yasin -->	  
+      <!-- END OF: Graduate Student: Kosar Yasin -->	
+
+	     <!-- Graduate Student: Layla Heidarizadeh-->
+      <div class="row">
+
+        <!-- Photo and name -->
+        <div class="col-4">
+          <h3>Ms. Kosar Yasin</h3>
+          <img style="width: 276px; height: 276px;" title="Layla Heidarizadeh" alt="Layla Heidarizadeh" src="group/Layla Heidarizadeh.jpg">

```

Comment:
make sure to add the picture file and name it such that it doesn't contain white spaces, also in the `src=` part of the code. Otherwise it may not work.